backend/shell.nix

```diff
@@ -2,8 +2,10 @@
 
 pkgs.mkShell {
   buildInputs = [
-    pkgs.go            # Go compiler
-    pkgs.git           # Git, in case you need it for version control
+    pkgs.go
+    pkgs.git
+    pkgs.nodejs
```

Comment:
You're adding node and npm to nix for the data dump?

Reply:
Yeah because I had to install npm to run elasticdump

Reply:
Mhmm but the problem is in sandbox, not backend right? Should we create a new shell.nix there instead?